toolbox/include/DotJNu.h

```diff
@@ -2,23 +2,33 @@
 #define WBT_DOTJDOTQ_H
 
 #include "WBIModelBlock.h"
+#include <iDynTree/Core/VectorFixSize.h>
 
 namespace wbt {
     class DotJNu;
 }
 
-class wbt::DotJNu : public wbt::WBIModelBlock {
+namespace iDynTree {
+    class Twist;
+    class Transform;
+    class VectorDynSize;
+    typedef int FrameIndex;
```

Comment:
I tried to keep as much as possible the same behavior as the current state, and I agree with you on these forward declaration. For `VectorFixSize` I had to add the header since redefining templates was a bit too much imo, but applying this line to all the typedefs would make sense too